WBooks/Repositories/BibliotecaRepository.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  BibliotecaRepository.swift
+//  WBooks
+//
+//  Created by Florencia Rosental on 02/03/2018.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Networking
+import Argo
+import Result
+import ReactiveSwift
+
+protocol BibliotecaRepositoryType {
+    
+    func fetchBooks() -> SignalProducer<[Book], RepositoryError>
+    
+}
+
+internal class BibliotecaRepository: AbstractRepository, BibliotecaRepositoryType {
+    
+    private static let EntitiesPath = "books"
+    private static let PageKey = "page"
+    
+    private static let ReadPrefixPath = "users"
+    private static let ReadSuffixPath = "notifications/read_all"
+    
+    private static let FirstPage = 1
+    
+    public func fetchBooks() -> SignalProducer<[Book], RepositoryError> {
+        let path = BibliotecaRepository.EntitiesPath
+        let parameters = [BibliotecaRepository.PageKey: BibliotecaRepository.FirstPage]
```

Comment:
If you do that you are only fetching the first page. Lets use `[]` as parameters.